python/cinder-understack/pyproject.toml

```diff
@@ -48,3 +49,6 @@ filterwarnings = "ignore::DeprecationWarning"
 # use our default and override anything we need specifically
 extend = "../pyproject.toml"
 target-version = "py310"
+
+[project.entry-points."cinder.volume.drivers"]
+netapp-understack = "cinder_understack.dynamic_netapp_driver:NetappCinderDynamicDriver"
```

Comment:
I think you won't be able to use a dash here. I think you'll have to name it something like `NetappCinderDynamicDriver`